### PR TITLE
Add theories for MC PDFs

### PR DESCRIPTION
**Pull request by scarlehoff**

### CODE REVIEW

**Comment by RoyStegeman** [comment]
Are you sure we should keep using the old theoryids as the second part?

**Comment by scarlehoff** [comment]
You are right, also the fact that they are "MC theories" is an accident. They are just the perturbative charm of 4.0 so it is not even 41.
I'll call them 40001000. (and 40000000 should be 700)

**Comment by scarlehoff** [comment]
The last commit adds keys necessary for pineko: https://github.com/NNPDF/pineko/issues/177

**Review by niclaurenti — APPROVED** [review]

File: nnpdf_data/nnpdf_data/theory_cards/41390102.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+ID: 41_390_102
+Comments: NNPDF40 for MC generators, LO QCD, LO QED
```
[the remark this answers is not in the document]
Comment (scarlehoff) [reply nnpdf_data/nnpdf_data/theory_cards/41390102.yaml]: The eko for this you created was with QED 1 or 2?

[the remark this answers is not in the document]
Comment (niclaurenti) [reply nnpdf_data/nnpdf_data/theory_cards/41390102.yaml]: QED=2

**Pull request merged**